airbyte-integrations/connectors/source-datadog/source_datadog/spec.yaml

```diff
@@ -0,0 +1,56 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/datadog
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Datadog Source Spec
+  type: object
+  required:
+    - api_key
+    - application_key
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API Key
+      description: Datadog API key
+      type: string
+      airbyte_secret: true
+      order: 1
+    application_key:
+      title: Application Key
+      description: Datadog application key
+      type: string
+      airbyte_secret: true
+      order: 2
+    query:
+      title: Query
+      description: The search query. This just applies from Logs and Audit Logs entities.
+      type: string
+      order: 3
+    limit:
+      type: integer
+      title: Limit
+      default: 5000
+      minimum: 1
+      maximum: 5000
+      description: Maximum number of records to collect per request.
+      order: 4
+    start_date:
+      title: Start date
+      pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
+      description: >-
+        UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will
+        not be replicated. This just applies from Logs and Audit Logs entities.
```

Comment:
```suggestion
        not be replicated. This just applies to Incremental syncs.
```
what do you think? if someone implement another incremental stream in the future it will continue to work.

Reply:
Great catch!